model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -203,34 +206,33 @@ def make_model_config(
         model_config_dict["name"] = variant_name
         logger.info("")
         if variant_found:
-            logger.info(f"Found existing model config: {model_config_dict['name']}")
+            logger.info(f"Found existing model config: {variant_name}")
         else:
-            logger.info(f"Creating model config: {model_config_dict['name']}")
+            logger.info(f"Creating model config: {variant_name}")
         for str in logger_str:
             logger.info(str)
         logger.info("")
 
         model_config = ModelConfig.create_from_dictionary(model_config_dict)
         model_config.set_cpu_only(model.cpu_only())
 
-        return model_config
+        return ModelConfigVariant(model_config, variant_name)
 
     @staticmethod
-    def make_ensemble_model_config(
+    def make_ensemble_model_config_variant(
         model: ModelProfileSpec,
-        ensemble_composing_model_configs: List[ModelConfig],
+        ensemble_composing_model_config_variants: List[ModelConfigVariant],
         model_variant_name_manager: ModelVariantNameManager,
         param_combo: Dict = {},
-    ) -> ModelConfig:
+    ) -> ModelConfigVariant:
         """
-        Loads the ensemble model spec from the model repository, and then mutates
-        the names to match the ensemble composing models
+        Loads the ensemble model spec from the model repository
```

Comment:
Is this comment true? Does it load the one from the repo? Or does it create a variant of it?